ci-operator/jobs/windup/windup-ui-tests/windup-windup-ui-tests-main-presubmits.yaml

```diff
@@ -0,0 +1,54 @@
+presubmits:
```

Comment:
How do we know when this pre-submit job will run?

Is there anyway that we can set this up using something like `has_changed`
https://steps.ci.openshift.org/ci-operator-reference#line831

So that it only runs a job on product qe's PRs when they are changing something that is relied on by the Dockerfile.

Reply:
Sounds good, I think this could be a case by case basis. The good thing is that its just building an image so its not a incurring any cloud costs or anything. Then the team also has the ability to /skip the test if they are doing something that doesn't effect the image build, though I'd advise against that.

If we find a very active repo where the layered product test image build is only a small subset of what they do we can look into this further.